tests/github/test_dal.py

```diff
@@ -0,0 +1,28 @@
+import os
+from unittest import mock
+
+from pytest_mock import MockerFixture
+
+from checkov.github.dal import Github
+
+
+@mock.patch.dict(os.environ, {"GITHUB_ORG": "simpleOrg"}, clear=True)
+def test_org_security(mocker: MockerFixture):
+    dal = Github()
+    mock_data = {
+        "data": {
+            "organization": {
+                "name": "Bridgecrew",
+                "login": "Bridgecrew-dev",
+                "description": "",
+                "ipAllowListEnabledSetting": "DISABLED",
+                "ipAllowListForInstalledAppsEnabledSetting": "DISABLED",
+                "requiresTwoFactorAuthentication": False,
+                "samlIdentityProvider": None
+            }
+        }
+    }
+    graphql_mock = mocker.MagicMock(return_value=mock_data)
+    mocker.patch("checkov.common.vcs.base_vcs_dal.BaseVCSDAL._request_graphql", graphql_mock)
```

Comment:
```suggestion
    mocker.patch("checkov.common.vcs.base_vcs_dal.BaseVCSDAL._request_graphql", return_value=mock_data)
```
if I'm not wrong, this should be enough 🙂  it makes sense to `MagicMock` when you actually want to validate that it was accessed or have more complex interactions with it.